wrappers/golang/curves/bls12377/g2/g2_msm.go

```diff
@@ -1,12 +1,12 @@
-//go:build g2
-
-package bls12377
+package g2
 
 // #cgo CFLAGS: -I./include/
 // #include "g2_msm.h"
 import "C"
 
 import (
+	. "github.com/ingonyama-zk/icicle/wrappers/golang/curves/bls12377"
```

Comment:
Why is this needed?

Reply:
it's needed to import `ScalarField` and its functions